datasets/ccaligned_multilingual/README.md

```diff
@@ -0,0 +1,321 @@
လိၵ်ႈ၊ ႁူင်းတူၺ်းလိၵ်ႈ ၸဝ်ႈၵၢင်းသိူဝ် ၶၢဝ်းတၢင်းတႄႇၶပ်ႉပပ်ႉလိၵ်ႈ တႄႇဝၼ်းထိ 10 – 20. 04 . 2018 ၼႆယူႇယဝ်ႉ။|ၸိူဝ်းပဵၼ်ၸဝ်ႈပီႈၸဝ်ႈၼွင်ႉ ၸိူဝ်းမီးၶၢဝ်းယၢမ်းတူဝ်ႈတၼ်းၸိူဝ်းၼၼ်ႉ လႄႈ မၢင်ၸိူဝ်းၵေႃႈ တတ်းဢဝ်ၶၢဝ်းယၢမ်း ၼႂ်းၵႄႈဢမ်ႇတၼ်းၼၼ်ႉသေ မႃးၸွႆႈမႃးထႅမ် မိူၼ်ၼင်ႇဝႃႈ ႁူမ်ႈၵၼ်ၵိၼ်ၸင်ႇဝၢၼ် ႁူမ်ႈၵၼ်ႁၢမ်ၸင်ႇမဝ် ၼႆၼၼ်ႉ ယဝ်ႉ။ တွၼ်ႈတႃႇၼိုင်ႈပီၼိုင်ႈပီၼႆႉ ဢၼ်ပဵၼ်ပပ်ႉလိၵ်ႈ ဢင်းၵိတ်ႉ၊ တႆး ၊ ထႆး၊ မၢၼ်ႈ တႄႇၵႂႃႇၸိူဝ်းၼႆႉၵေႃႈ တိူဝ်းၼမ် မႃးတိၵ်းꧦ ၵူႈပီပီလႄႈ လႆႈ Update ပၼ်သဵၼ်ႈမၢႆမၼ်ႈယူႇ ၵူႈပီပီၼႆယဝ်ႉ။ ဢၼ်ၼမ်လိူဝ်ၼႆႉတႄႉ တေပဵၼ်ဢင်းၵိတ်ႉ ယဝ်ႉၶႃႈ၊ ယွၼ်ႉပိူဝ်ႈဝႃႈ ဢင်းၵိတ်ႉၼႆႉ ပဵၼ်လၵ်းထၢၼ် ၵၢၼ်ႁဵၼ်း တွၼ်ႈတႃႇ လုၵ်ႈႁဵၼ်းသင်ႇၶတႆး ၸိူဝ်းဢၼ်မႃး ၶိုၼ်ႈႁဵၼ်း လဵပ်ႈႁဵၼ်း ပႆၸွမ်းၶၢဝ်းတၢင်းသီႇႁူဝ်ႇၼႆႉ|ၼႆလႄႈ ၸဝ်ႈၶူးလူင် Prof. Dr. ၶမ်းမၢႆ ထမ်မသႃမိ ၸင်ႇဢွၼ်ႁူဝ်တႄႇတင်ႈ မႃးပၼ်ႁဝ်းၶႃႈ ၸဝ်ႈပီႈၸဝ်ႈၼွင်ႉ ႁႂ်ႈပေႃးလႆႈတူၺ်းလႆႈ တွင်းလႆႈ ႁႃႈလႆႈၶေႃႈမုၼ်း ၵဵဝ်ႇလူၺ်ႈၵၢၼ်ႁဵၼ်း၊ ဢၼ်ပဵၼ် တၢင်းၸွႆႈထႅမ်လွင်ႈၵၢၼ်ႁဵၼ်းလႆႈ ငၢႆႈငၢႆႈၼႆၼၼ်ႉယူႇယဝ်ႉ၊ ၼႆလႄႈ ၸိူဝ်းပဵၼ်လုၵ်ႈၼွင်ႉၸဝ်ႈၶူးလူင်ၵေႃႈ ၸင်ႇလႆႈသိုပ်ႇ ထိင်းသိမ်း၊ သိုပ်ႇႁဵၼ်းၵၢၼ်ၵႂႃႇ ပၢၼ်သိုပ်ႇပၢၼ်ယူႇၶႃႈယဝ်ႉ။|မိူဝ်ႈဝၼ်းထိ 08.06.2017 ပၢင်ႁူပ်ႉထူပ်း ၸုမ်းဝႆႈၽြႃး ဢၼ်ၸဝ်ႈၶူး မုၼိဝရ (လၢႆးၶႃႈ) လႄႈ ၸဝ်ႈၶူး ၺႃၼဝရ (ၵျွၵ်းမႄး) ဢွၼ်ႁူဝ် တၵ်ႉၵႃသထႃး ဝဵင်းၵျွၵ်းမႄး၊ ဝဵင်းလၢႆးၶႃႈ လႄႈ သင်ႇၶတႆး မိူင်းသီႇႁူဝ်ႇ။|ယိူင်းၸူးလူႇတၢၼ်းပၼ် ၶိင်းသူၺ်ႇၵျွင်းဢၢႆႈၽူင်း ၼိုင်ႈလိူၼ်တဵမ် (AD. 1943-2017)|(ၽူႈဢၼ်လူႇတၢၼ်းယႂ်ႇလူင် ၼႂ်းၽႃႇသႃႇ၊ သႃႇသၼႃႇ၊ လႄႈ ၼႃႈယၵ ၵေႃလိၵ်ႈလၢႆးလႄႈ ၽင်ႈ|ငႄႈတႆး ဝဵင်းလိူဝ်ႇ) ဢၼ်သဵင်ႈၵႂႃႇ တဵမ်ၼိုင်ႈလိူၼ်သေ ၼၢႆးသူၺ်ႇၵျွင်းဢီႇသၢဝ် လႄႈ|လုၵ်ႈလၢၼ်တင်းသဵင်ႈ (မူႇၸေႈ၊ လႃႈသဵဝ်ႈ၊ တႃႈလိူဝ်ႇ၊ တႃႈၵုင်ႈ) လူႇတၢၼ်းၵၢပ်ႈသွမ်းဢေႃႈ။ (10/06/2017)|သွၵ်ႈႁႃ|Search for:|Search|ၽႃႇသႃႇတႆး:|English|Shan|ၿုၻ်ꩪၸဝ်ႈ|ပၵ်းယဵမ်ႈဝၼ်း|December 2019|M|T|W|T|F|S|S|« Sep|1|2 3 4 5 6 7 8|9 10 11 12 13 14 15|16 17 18 19 20 21 22|23 24 25 26 27 28 29|30 31|ၾဵတ်ႉၶၢဝ်းတၢင်းသီႇႁူဝ်ႇ|ပပ်ႉၸဝ်ႈၶူး Dr. မႁေႃသထႃလင်ၵႃရႃၽိဝမ်သ|ၶၢဝ်ႇမိူဝ်ႈလဵဝ်|ႁၢင်ႈၽၢင်မၢႆတွင်း ႁပ်ႉၸုမ်ႈၶူး M.A & P.G.D တီႈၸၼ်ႉၸွမ် ၵေႇလၼိယ လႄႈ ပၢင်ႁူပ်ႉထူပ်းလုၵ်ႈႁဵၼ်းသင်ႇၶတႆး မိူင်းသီႇႁူဝ်ႇ|ပွင်ႇလႅင်းထိုင် ၽူႈႁၵ်ႉလိၵ်ႈလၢႆးပၢႆပႄႇႁဝ်းၶဝ်တင်းသဵင်ႈတီႈၶႃႈ ဢမ်ႇပေႃးႁိုင်ပေႃးၼၢၼ်း ဢၼ်တေပဵၼ်ပပ်ႉမၢႆတွင်း ပီၵွၼ်းၶမ်း (50) ပီတဵမ်|ယွၼ်းမူႇလိၵ်ႈၶၢဝ်းတၢင်းသီႇႁူဝ်ႇ မၢႆ-21 ၊ ပီ – 2019 ထိုင် ၸဝ်ႈပီႈၸဝ်ႈၼွင်ႉႁဝ်းၶဝ်တင်းသဵင်ႈ|လုၵ်ႈႁဵၼ်းသင်ႇၶတႆး လႆႈၸုမ်ႈ M.A တီႈၸၼ်ႉၸွမ် Buddhist & Pāli Unversity မိူင်းသီႇရိလင်းၵႃ၊ သီႇၸဝ်ႈ|သဵၼ်ႈမၢႆၸဝ်ႈပီႈၸဝ်ႈၼွင်ႉ ၸိူဝ်းဢွင်ႇပူၼ်ႉၸၼ်ႉလိၵ်ႈၼႂ်းပီ 2018|ၶေႃႈပၼ်တၢင်းၶႆႈၸႂ်|ႁွင်ႈၶၢဝ်ႇ|September 2019|August 2019|June 2019|February 2019|December 2018|November 2018|October 2018|September 2018|August 2018|July 2018|May 2018|April 2018|March 2018|December 2017|November 2017|October 2017|September 2017|August 2017|July 2017|June 2017|May 2017|မူႇၵေႃ|ၵၢၼ်တူင်ႉၼိုင်|ၶၢဝ်ႇၼႂ်းမိူင်း|News|Uncategorized|ၶၢဝ်ႇၼွၵ်ႈမိူင်း|Meta|Log in|Entries RSS|Comments RSS|WordPress.org|တူဝ်ၼပ်ႉၵူၼ်းမႃးယဵမ်ႈ|Copyright 2015 © 2017 Cjtaisangha.com |Makutarama Temple 42/15 Reservoir Road, Dematagoda, Colombo 09, Sri Lanka Tel: 009 411 2662488 Email: Lankajourney@yahoo.co.uk |Ribosome by GalussoThemes.com|Powered by WordPress|\n', 'Target_URL': 'http://www.cjtaisangha.com/activities/'}
```

Comment:
Can you update this example now that we're using the `Translation` feature type please ?

Reply:
Yes. I missed the README. Sorry.